platform/android/java_class_wrapper.cpp

```diff
@@ -132,7 +132,7 @@ bool JavaClass::_call_method(JavaObject *p_instance, const StringName &p_method,
 			if (arg_expected != Variant::NIL) {
 				r_error.error = Callable::CallError::CALL_ERROR_INVALID_ARGUMENT;
 				r_error.argument = i;
-				r_error.expected = arg_expected;
+				r_error.expected = arg_expected; // ?
```

Comment:
```suggestion
				r_error.expected = arg_expected;
```

I got it, there is no mistake. The variable should probably be renamed or the code rearranged.